build.gradle

```diff
@@ -20,5 +21,20 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven { url "https://www.jitpack.io" }
     }
 }
+
+ext {
+    // Sdk and tools
+    minSdkVersion = 11
+    targetSdkVersion = 25
```

Comment:
>  We can move that to API 16 if required.

No it's perfect like this, the older the better. I was just not expected 11 to be possible so tough it was about something else :)